lib/inspec/profile.rb

```diff
@@ -239,6 +239,15 @@ def info(res = params.dup) # rubocop:disable Metrics/CyclomaticComplexity
         data[:impact] = 1.0 if data[:impact] > 1.0
         data[:impact] = 0.0 if data[:impact] < 0.0
         data[:id] = id
+
+        # if the code field is empty try and pull info from dependencies
+        if data[:code].empty? && parent_profile.nil?
+          locked_dependencies.dep_list.each do |_name, dep|
+            profile = dep.profile
+            code = Inspec::MethodSource.code_at(data[:source_location], profile.source_reader)
+            data[:code] = code unless code.nil? || code.empty?
```

Comment:
I may just not be following, but won't this get overwritten by the last entry in `dep_list`? Is that the intended behavior?

Reply:
Yeah we need a return